README.md

```diff
@@ -145,25 +145,34 @@ export class Controller {
 
 }
 ```
+
 **returntype.model.generated.ts:**
 ```typescript
 export interface ReturnType {
     method: string;
 }
 ```
 
-**api.module.ts:**
+**serviceconfig.ts:**
 ```typescript
-import { NgModule, ModuleWithProviders, Injectable } from '@angular/core';
+import { Injectable } from '@angular/core';
+import { HttpErrorResponse } from '@angular/common/http';
 import { Observable } from 'rxjs/Observable';
-import { Controller } from './controller.generated';
 
 @Injectable()
 export abstract class ServiceConfig {
     context?: string;
     debug?: boolean;
-    onError?(): Observable<any>;
+    onError?(error:? HttpErrorResponse): Observable<any>;
```

Comment:
This should probably be _error?:_ instead of _error:?_

Reply:
My bad.  thanks for catching.  Fixed in #75